db.js

```diff
@@ -256,6 +262,41 @@ exports.init = function (sbot, dir, config) {
     indexes[index.name] = index
   }
 
+  function updateIndexes() {
+    const start = Date.now()
+
+    const indexesRun = Object.values(indexes)
+    let isLive = false
+
+    function liveStream() {
+      debug('live streaming changes')
+      isLive = true
+      log.stream({ gt: indexes['base'].seq.value, live: true }).pipe({
+        paused: false,
+        write: (data) => indexesRun.forEach((x) => x.onData(data, isLive)),
```

Comment:
I was looking back and forth at `isLive`, since it's defined up there, mutated here, used down there, etc, and I think now that I understand this code, it would have been easier to understand the code without this variable.

Just call `x.onData(data, true)` or `x.onData(data, false)` here and there, then we can avoid the variable and its mutation and mentally following how it's updated.

Reply:
Thanks